orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/ConfigurationService.java

```diff
@@ -414,6 +414,10 @@ public Optional<String> getIPVCapacity() {
         }
     }
 
+    public boolean setAuthenticatedFlagForIpv() {
```

Comment:
Bit of a nitpick but reads a bit odd to have a method named `set` which actually does getting. Going to be removed soon anyway so not a blocker to merge

Reply:
Can rename to `IsAuthenticatedFlagForIpvEnabled`

Reply:
Have renamed to the suggested